examples/official-storybook/tests/addon-jest.config.json

```diff
@@ -0,0 +1,19 @@
+{
+  "rootDir": "../../..",
+  "verbose": true,
+  "coverageDirectory": "coverage/",
+  "notify": true,
+  "transform": {
+    ".*": "babel-jest"
+  },
+  "snapshotSerializers": [
+    "enzyme-to-json/serializer"
+  ],
+  "setupFiles": ["raf/polyfill"],
+  "setupTestFrameworkScriptFile": "<rootDir>/scripts/jest.init.js",
+  "moduleNameMapper": {
+    "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>__mocks__/fileMock.js",
+    "\\.s?css$": "<rootDir>__mocks__/styleMock.js"
+  },
+  "testMatch": ["<rootDir>/examples/official-storybook/tests/addon-jest.test.js"]
```

Comment:
On master, there're also some coverage options specified: https://github.com/storybooks/storybook/blob/master/examples/official-storybook/config-addon-jest.json#L18-L27

Reply:
Is this required for addon-jest to work ? I was thinking maybe this was not used after all ... Will re-add them.